Acceptance test exit feature

momeyer opened the pull request:
Now we can include our files using <>.
Changed python file name.
Added workflow to run acceptance test on circleci, now it works fine.

Ryno95 commented on src/parser/dispatcher.c:
```diff
@@ -1,14 +1,11 @@
 {
 	(void)input;
 	if (command == EXIT)
-		return (exit_command());
```
Fair enough, no use in returning when exiting :)

Ryno95 reviewed (approved):
Looks good, basically some clean-up and the addition of the acceptance test for exit() that we did at kanteen yesterday right?

momeyer commented:
Exactly.
I noticed that we wrote the test case for the dispatcher with the EXIT command, but it was just exiting so we were not even getting the return value of the function. So I changed the test case and also removed the return(exit_command()); because it doesn't make sense haha
